src/config_manager.sh

```diff
@@ -0,0 +1,69 @@
+declare -A paths
+declare -A comments
+
+DB_PATH=$DEFAULT_CONFIG_PATH/configdb
+
+function load_configs()
+{
+  local i=0
+
+  while read line
+  do
+    if echo $line | grep -F = &>/dev/null
+    then
+      i=$(($i+1))
+      paths[$i]=$(echo $line | cut -d '=' -f 1 | tr -d '[:space:]')
+      comments[$i]=$(echo "$line" | cut -d '=' -f 2-)
+    fi
+  done < $DB_PATH
+}
+
+function register_config()
+{
+  load_configs
+
+  if [ $# -eq 0 ] ; then
```

Comment:
You have to be pickier in this validation, notice that verification pass if I just specify one or n elements (I suppose that you do not want it).